src/sulogin-shell/sulogin-shell.c

```diff
@@ -81,9 +81,10 @@ static void fork_wait(const char* const cmdline[]) {
 }
 
 static void print_mode(const char* mode) {
-        printf("You are in %s mode. After logging in, type \"journalctl -xb\" to view\n"
-                "system logs, \"systemctl reboot\" to reboot, \"systemctl default\" or ^D to boot\n"
-                "into default mode.\n", mode);
+        printf("You are in %s mode. If the root account is enabled, you must log in\n"
+               "as root after this message. In the shell, type \"journalctl -xb -p3\" to view\n"
```

Comment:
i think we should use `-perr` rather than `-p3` here, as that's more descriptive, and people looking at the command line might more likely be able to understand what's being done here...

Reply:
I'll do that.

Reply:
Wouldn't it be wiser to tell people to do "systemctl --failed" and "systemctl status <unit>" ?

Most of the time, when I end up in emergency mode, journald isn't running yet and has little info. systemctl is where the real diagnosis starts...